FlightPlannerD/DbContext/EntityExtensions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace FlightPlannerD.DbContext
+{
+   
+        public static class EntityExtensions
+        {
+            public static void Clear<T>(this DbSet<T> dbSet) where T : class
```

Comment:
šito vispār kaut kur izmanto?

Reply:
Nē, tika mēģināts pievienot funkcionalitāti Clear, bet nesanāca. Savādāk apiets. Izdzēsts.